### PR TITLE
Added platform switch for Dragonboard410c

### PR DESCRIPTION
**Pull request by psivasubramanian**
Switch to exercise hardware acceleration and DMAbuf path on DB410c

### CODE REVIEW

File: recipes-wpe/wpewebkit/wpewebkit_0.1.bb [hunk 1]
```diff
@@ -52,6 +52,7 @@ PACKAGECONFIG[westeros-mesa] = "-DUSE_WPEWEBKIT_BACKEND_WESTEROS_MESA=ON,,"
 # WPE Platform specific switches
 PACKAGECONFIG[intelce] = "-DUSE_WPEWEBKIT_PLATFORM_INTEL_CE=ON -DUSE_HOLE_PUNCH_GSTREAMER=ON,,"
 PACKAGECONFIG[nexus] = "-DUSE_WPEWEBKIT_PLATFORM_BCM_NEXUS=ON -DUSE_HOLE_PUNCH_GSTREAMER=ON,,"
+PACKAGECONFIG[qcomdb] = "-DUSE_WPEWEBKIT_PLATFORM_DB410C=ON,,"
```
Comment (psivasubramanian) [inline recipes-wpe/wpewebkit/wpewebkit_0.1.bb]: Currently only DB410C is using this switch and I hope the same can be used for similar board DB820c also in the future.

**Comment by kraj** [comment]
At this time it's unknown if we will need changes for 820 or not

**Comment by wouterlucas** [comment]
Exactly, so lets keep it in line with the Broadcom/Intel/etc switch, which is generic for all their models. I'd like to see the WPE -D flag generic to Qualcomm.

**Comment by psivasubramanian** [comment]
Hi @wouterlucas, addressed your review comment.

**Pull request merged**